database/migrations/2021_11_29_151352_announcement.php

```diff
@@ -15,9 +15,9 @@ public function up()
     {
         Schema::create('announcements', function(Blueprint $table) {
             $table->id();
-            $table->text('title');
+            $table->string('title');
             $table->text('message');
-            $table->string('status');
+            $table->enum('status', ['active', 'inactive'])->default('active');
```

Comment:
Default needs to be inactive and the scheduled job makes it active when the selected time frame is reached